pkgs/development/coq-modules/odd-order/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, mkCoqDerivation, coq, mathcomp, version ? null }:
```

```suggestion
{ lib, mkCoqDerivation, mathcomp, version ? null }:
```

The `coq` parameter is not used.